src/libdvdetect.mk

```diff
@@ -0,0 +1,32 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := libdvdetect
+$(PKG)_IGNORE   := 
+$(PKG)_VERSION  := 0.71.0
+$(PKG)_CHECKSUM := b098e04660532df78836f50bc0a8044b66c6659b07a6bff6609724ad30a87192
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/nschlia/libdvdetect/releases/download/RELEASE_0_71/$(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc tinyxml
```

Comment:
(Not to fix in this pull request.)
The build prints `tinyxml is deprecated - please use tinyxml2`
Upgrading to tinyxml2 requires code changes in libdvdetect.
I think, it makes sense to do it in https://github.com/nschlia/libdvdetect and then update MXE version of libdvdetect replacing tinyxml dependency with tinyxml2.